playground/mocks/config/responseConfig.js

```diff
@@ -145,6 +147,9 @@ const idx = {
     // 'authenticator-verification-okta-verify-push',
     // 'authenticator-verification-custom-app-push',
     // 'authenticator-verification-custom-app-push-reject',
+    // 'authenticator-enroll-ov-sms-enable-biometrics',
+    // 'okta-verify-version-upgrade',
+    'okta-verify-uv-verify-enable-biometrics'
```

Comment:
This should be the new default?

Reply:
there such no defaults for the mocks.. but i will change.